modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java

```diff
@@ -461,12 +461,16 @@ public List<SystemView<?>> systemViews() {
 
     class OnUpdateHandlerImpl implements OnUpdateHandler {
         @Override
-        public CompletableFuture<Void> handle(UpdateLogEvent event, HybridTimestamp metaStorageUpdateTimestamp, long causalityToken) {
+        public CompletableFuture<Void> handle(MarshallableEntry event, HybridTimestamp metaStorageUpdateTimestamp, long causalityToken) {
```

Comment:
Previous signature was better. The method doesn't accepts arbitrary MarshallableEntry, but only top-level entries: Snapshot and VersionedUpdate